Spigot-Server-Patches/0424-Added-EndGatewayCreationEvent.patch

```diff
@@ -47,7 +47,7 @@ index 09eabf123..87e6b63e5 100644
 +            Location loc = MCUtil.toLocation(this.getWorldServer(), j, 75, k);
 +            EndGatewayCreationEvent event = new EndGatewayCreationEvent(loc.getBlock());
 +            Bukkit.getPluginManager().callEvent(event);
-+            if (event.isCancelled()) {
++            if (!event.callEvent()) {
```

Comment:
Maybe use `if(!new com.destroystokyo.paper.event.block.EndGatewayCreationEvent(loc.getBlock()).callEvent())` instead to reduce diff to smallest possible?

Reply:
You're essentially double firing the event here. standard Bukkit callevent and this callevent. Do as mikro suggested.